index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>David Isea Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
```

Comment:
@AlucardSanin Kindly adhere to the Figma template design you have chosen. Below is the screenshot of the correct design

![alucardcorrect](https://user-images.githubusercontent.com/87413774/166082079-6f73f272-bdbc-4845-93dd-ddbb9173f50d.png)

Below is the screenshot of your design

![alucardwrong](https://user-images.githubusercontent.com/87413774/166082126-79905217-b459-4de1-a89f-0d394613cef7.png)

Reply:
Ok will try to put it more alike, it's just that it doesn't show to me like that, but that's probably something related to my monitor resolution on the web browser, this is who it is shown to me:

![2022-04-29 (1)](https://user-images.githubusercontent.com/99287916/166082570-c299dc23-ff3d-4774-8760-ea0d7533d573.png)